docs/for-ops/how-to/manage-age.md

```diff
@@ -0,0 +1,162 @@
+---
+slug: manage-age
+title: Manage Age for Development, Security, and Recovery
+sidebar_label: Manage Age
+---
+
+:::info
+This documentation was written for SOPS that is using Age Encryption. Please check the SOPS page for other [KMS](/docs/get-started/installation/sops#use-sops-with-an-external-key-management-service-kms) providers.
+:::
+
+## Prerequisites
+
+- Version >= `v3.1`
+
+## Local development setup
+
+### Steps
+
+1. Install [Age](https://github.com/FiloSottile/age/):
+
+```bash
+brew install age
+```
+
+2. Verify Age Versions:
+
+```bash
+age --version
+age-keygen --version
+```
+
+3. Install [Dependencies](https://github.com/linode/apl-core/blob/main/bin/install-deps.sh):
+
+```bash
+helm plugin install https://github.com/databus23/helm-diff.git
+helm plugin install https://github.com/jkroepke/helm-secrets.git --version v3.15.0
+```
+
+4. Create a values folder:
+
+```bash
+mkdir -p values-folder ~/workspace
+```
+
+5. Clone the values repo from Gitea:
+
+:::info
+Don't forget to set GIT_SSL_NO_VERIFY!
```

Comment:
In case your TLS certificate is not trusted, set GIT_SSL_NO_VERIFY to clone the git repository.